### ISSUE TITLE
Raspberry Pi

### ISSUE THREAD

**Issue by angelikalabininay** [Asadullah-Dal17/Distance_measurement_using_single_camera#21]
Hello! I'd like to ask if you have tried using this on Raspberry Pi? Would it work?

**Comment by Asadullah-Dal17** [comment]
Yeah definitely it will work on Raspberry pi, You have to make some changes, 
check out his repo for [face detection on pi4 ](https://github.com/Asadullah-Dal17/face-detection-Comparsion-between-Opencv-and-Dlib-on-Raspberry-Pi-4)
